src/sagemaker_core/tools/additional_operations.json

```diff
@@ -171,6 +171,16 @@
             "service_name": "sagemaker-metrics"
         }
     },
+    "Metrics": {
+        "BatchGetMetrics": {
+            "operation_name": "BatchGetMetrics",
+            "resource_name": "Metrics",
```

Comment:
The `resource_name` should be `TrialComponent` and the `method_type ` should be `class`